docroot/themes/humsci/su_humsci_theme/scss/base/_global.scss

```diff
@@ -8,6 +8,57 @@
 // SU HumSci
 @import '../utilities/mixins/index';
 
+// Need to ensure that the new WYSIWYG text-editor classes are supported with the
+// update made by the Sparkbox team to the Humsci Basic Theme
+// For example, instead of .decanter-font-splash, you can now use
+// .hs-font-splash as a more unspecific override of WYSIWYG styles
+.hs-external-link {
+  @include external-link(external-link, external-link-hover);
```

Comment:
@silverli, I think I'm seeing an issue here where the file path to the image isn't correct anymore:

![image](https://user-images.githubusercontent.com/1256329/78270210-87e65a80-74d8-11ea-99c6-764f7e6f36bb.png)

You mentioned this in the summary at https://github.com/SU-HSDO/suhumsci/pull/540, so maybe you can just show me what I should be expecting to see?

Reply:
I chatted with Silver... it looks like these assets just got left out of https://github.com/SU-HSDO/suhumsci/pull/540/files somehow. I went ahead and re-added them in a new commit and now this is working as expected. 👍